types.d.ts

```diff
@@ -184,4 +205,42 @@ declare module 'binance-api-node' {
             locked: string;
         };
     }
+
+    interface OutboundAccountInfo extends Message {
+        balances: Balances;
+        makerCommissionRate: number;
+        takerCommissionRate: number;
+        buyerCommissionRate: number;
+        sellerCommissionRate: number;
+        canTrade: boolean;
+        canWithdraw: boolean;
+        canDeposit: boolean;
+        lastAccountUpdate: number;
+    }
+
+    interface ExecutionReport extends Message {
+        symbol: string;

```

Comment:
Let's sort all these attributes alphabetically, otherwise it's hard to maintain them.

Reply:
++